examples/ios/swift/Migration/AppDelegate.swift

```diff
@@ -67,77 +29,50 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
-                        // combine name fields into a single field
-                        let firstName = oldObject!["firstName"] as! String
-                        let lastName = oldObject!["lastName"] as! String
-                        newObject?["fullName"] = "\(firstName) \(lastName)"
-                    }
-                }
-            }
-            if oldSchemaVersion < 2 {
-                migration.enumerateObjects(ofType: Person.className()) { _, newObject in
-                    // give JP a dog
-                    if newObject?["fullName"] as? String == "JP McDonald" {
-                        let jpsDog = migration.create(Pet.className(), value: ["Jimbo", "dog"])
-                        let dogs = newObject?["pets"] as? List<MigrationObject>
-                        dogs?.append(jpsDog)
-                    }
-                }
-            }
-            print("Migration complete.")
-        }
-
-        Realm.Configuration.defaultConfiguration = Realm.Configuration(schemaVersion: 3, migrationBlock: migrationBlock)
-
-        // print out all migrated objects in the default realm
-        // migration is performed implicitly on Realm access
-        print("Migrated objects in the default Realm: \(try! Realm().objects(Person.self))")
-
-        //
-        // Migrate a realms at a custom paths
-        //
-        if let v1URL = bundleURL("default-v1"), let v2URL = bundleURL("default-v2") {
-            let realmv1URL = defaultParentURL.appendingPathComponent("default-v1.realm")
-            let realmv2URL = defaultParentURL.appendingPathComponent("default-v2.realm")
+        return true
+    }
 
-            let realmv1Configuration = Realm.Configuration(fileURL: realmv1URL, schemaVersion: 2, migrationBlock: migrationBlock)
-            let realmv2Configuration = Realm.Configuration(fileURL: realmv2URL, schemaVersion: 3, migrationBlock: migrationBlock)
+    private func addExampleDataToRealm(_ exampleData: (Realm) -> Void) {
+        let url = realmUrl(for: schemaVersion, usingTemplate: false)
+        let configuration = Realm.Configuration(fileURL: url, schemaVersion: UInt64(schemaVersion))
+        let realm = try! Realm(configuration: configuration)
 
-            do {
-                try FileManager.default.removeItem(at: realmv1URL)
-                try FileManager.default.copyItem(at: v1URL, to: realmv1URL)
-                try FileManager.default.removeItem(at: realmv2URL)
-                try FileManager.default.copyItem(at: v2URL, to: realmv2URL)
-            } catch {}
+        try! realm.write {
+            exampleData(realm)
+        }
+    }
 
-            // migrate realms at realmv1Path manually, realmv2Path is migrated automatically on access
-            try! Realm.performMigration(for: realmv1Configuration)
+    // Any version before the current versions will be migrated to check if all version combinations work.
+    private func performMigration() {
+        for realmVersion in 0..<schemaVersion {
```

Comment:
should avoid calling this `realmVersion`, since most people think of realm version as the version of the SDK, not the data base. Would consider `oldSchemaVersion`